src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -693,6 +694,16 @@ private void RegisterApplicationLifetimeEvents()
             _applicationLifetime.ApplicationStopping.Register(() =>
             {
                 Interlocked.Exchange(ref _applicationStopping, 1);
+                if (_environment.IsDrainOnApplicationStopping())
+                {
+                    var drainModeManager = _host?.Services.GetService<IDrainModeManager>();
+                    if (drainModeManager != null)
+                    {
+                        _logger.LogDebug("Application Stopping: enabling drain mode");
+                        drainModeManager.EnableDrainModeAsync(CancellationToken.None).Wait();
+                    }
+                    _logger.LogDebug("Draining completed");
```

Comment:
This log statement is misleading. At this point all we have done is called stop on all listeners. Which puts the function host in "drain mode". This should say something like "Drain mode enabled"